cfn-resources/encryption-at-rest/README.md

```diff
@@ -2,17 +2,65 @@
 
 ## Description
 This resource allows administrators to enable, disable, configure, and retrieve the configuration for Encryption at Rest.
```

Comment:
```suggestion
Returns and edits the Encryption at Rest using Customer Key Management configuration
```
can we make sure to leverage the Open API docs as much as possible for these definitions? https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Encryption-at-Rest-using-Customer-Key-Management

Reply:
This issue is fixed